sql/core/src/main/resources/org/apache/spark/sql/execution/ui/static/spark-sql-viz.js

```diff
@@ -210,8 +208,8 @@ function postprocessForAdditionalMetrics() {
   checkboxNode.click(function() {
       onClickAdditionalMetricsCheckbox($(this));
   });
-  var isChecked = window.localStorage.getItem("stageId-and-taskId-checked") == "true";
-  $("#stageId-and-taskId-checkbox").prop("checked", isChecked);
+  var isChecked = window.localStorage.getItem("stageId-and-taskId-checked") === "true";
+  checkboxNode.prop("checked", isChecked);
```

Comment:
Nice catch. Thanks!

Reply:
Is this a bug fix that needs to be backported?

Reply:
I think it's not a bug but improvement to avoid `$("#stageId-and-taskId-checkboc")` twice.
This is introduced by #28039 and merged into `master` and `branch-3.0` so I think it's better to backport this change into `branch-3.0`.

Reply:
Yes it's an improvement